Sources/Valet/Accessibility.swift

```diff
@@ -84,18 +84,4 @@ public enum Accessibility: Int, CustomStringConvertible, Equatable {
         
         return accessibilityAttribute as String
     }
-    
-    // MARK: Internal
-    
-    internal static func allValues() -> [Accessibility] {
```

Comment:
This was replaced by `CaseIterable`'s `allCases`. The downside of using `CaseIterable` is that the conformance is `public`. However, not having to maintain these values over time is also a plus. Happy to backtrack on this change (and the similar changes below). Thoughts?